src/domain/entities/DataPackage.ts

```diff
@@ -32,6 +33,7 @@ export interface DataPackageData {
         latitude: string;
         longitude: string;
     };
+    geometry?: Geometry;
```

Comment:
Not sure if this is something we discussed in another PR. Typically, we prefer to use: `geometry: Maybe<Geometry>`. This forces users of this datastructure to be explicit, do they need it or not? this is not a universal rule, for example options object of methods or props, sometimes makes no sense not to force the caller to put empty values. 

So up to you, but I wanted to mention it.

Reply:
Yes, let's keep it `optional?`. Just a note for future code, in general, code with discriminated unions prevent this kind of optional, as each type has exactly the fields it supports.